Subject: Various additions and updates to the device db.
 upstream commits:
  - 9c6f0d2a2b0f51da638dafd6115f744007688d77
  - c134502c5105fadd48bb7b60ba59b364e98ea6a1
  - 68f3cb6bcc0a6f24e66377a6f0ab707ff3457bf9
  - e9082f609faa9fe1301a86a52aca6563a1040e70
  - f9b50b4b6e7721c7d77e0f22779276c3a0981ad0
Origin: upstream
---
 src/music-players.h |   23 +++++++++++++++++++----
 1 file changed, 19 insertions(+), 4 deletions(-)

--- libmtp.orig/src/music-players.h
+++ libmtp/src/music-players.h
@@ -792,6 +792,9 @@
   { "Toshiba", 0x0930, "Gigabeat MET401", 0x001d,
       DEVICE_FLAG_NO_RELEASE_INTERFACE |
       DEVICE_FLAG_BROKEN_SEND_OBJECT_PROPLIST },
+  // Reported by Andree Jacobson <nmcandree@users.sourceforge.net>
+  { "Toshiba", 0x0930, "Excite AT300", 0x0963,
+      DEVICE_FLAGS_ANDROID_BUGS },
   // Reported by Nigel Cunningham <nigel@tuxonice.net>
   // Guessing on Android bugs
   { "Toshiba", 0x0930, "Thrive AT100/AT105", 0x7100,
@@ -1472,6 +1475,10 @@
   // Reported by equaeghe <equaeghe@users.sourceforge.net>
   { "SONY", 0x0fce, "Xperia U ST25i (MTP+ADB mode)", 0x5171,
       DEVICE_FLAGS_ANDROID_BUGS },
+  { "SONY", 0x0fce, "Xperia P (MTP+ADB mode)", 0x5172,
+      DEVICE_FLAGS_ANDROID_BUGS },
+  { "SONY", 0x0fce, "Xperia Sola MT27i (MTP+MSC+? mode)", 0xa173,
+      DEVICE_FLAGS_ANDROID_BUGS },
   // Reported by Anonymous Sourceforge user
   { "SonyEricsson", 0x0fce,  "j10i (Elm)", 0xd144,
       DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST },
@@ -1501,6 +1508,8 @@
       DEVICE_FLAGS_ANDROID_BUGS },
   { "Motorola", 0x22b8, "Xoom 2 Media Edition", 0x4311,
       DEVICE_FLAGS_ANDROID_BUGS },
+  { "Motorola", 0x22b8, "XT912", 0x4362,
+      DEVICE_FLAGS_ANDROID_BUGS },
   // Reported by Marcus Meissner to libptp2
   { "Motorola", 0x22b8, "IdeaPad K1", 0x4811,
       DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST },
@@ -1535,6 +1544,8 @@
   // Reported by anonymous Sourceforge user
   { "Google Inc (for Asus)", 0x18d1, "TF101 Transformer", 0x4e0f,
       DEVICE_FLAGS_ANDROID_BUGS },
+  { "Google Inc (for Samsung)", 0x18d1, "Nexus S", 0x4e21,
+      DEVICE_FLAGS_ANDROID_BUGS },
   // WiFi-only version of Xoom
   // See: http://bugzilla.gnome.org/show_bug.cgi?id=647506
   { "Google Inc (for Motorola)", 0x18d1, "Xoom (MZ604)", 0x70a8,
@@ -1702,8 +1713,9 @@
    */
   // Reported by anonymous SourceForge user
   { "Huawei", 0x12d1, "Honor U8860", 0x1051, DEVICE_FLAGS_ANDROID_BUGS },
+  { "Huawei", 0x12d1, "Mediapad (mode 0)", 0x360f, DEVICE_FLAGS_ANDROID_BUGS },
   // Reported by Bearsh <bearsh@users.sourceforge.net>
-  { "Huawei", 0x12d1, "Mediapad", 0x361f, DEVICE_FLAGS_ANDROID_BUGS },
+  { "Huawei", 0x12d1, "Mediapad (mode 1)", 0x361f, DEVICE_FLAGS_ANDROID_BUGS },
 
   /*
    * ZTE
@@ -1718,7 +1730,9 @@
   { "HTC", 0x0bb4, "Zopo ZP100", 0x0c02,
       DEVICE_FLAGS_ANDROID_BUGS },
   // Reported by Steven Eastland <grassmonk@users.sourceforge.net>
-  { "HTC", 0x0bb4, "EVO 4G LTE", 0x0ca8,
+  { "HTC", 0x0bb4, "EVO 4G LTE", 0x0c93,
+      DEVICE_FLAGS_ANDROID_BUGS },
+  { "HTC", 0x0bb4, "EVO 4G LTE (second ID)", 0x0ca8,
       DEVICE_FLAGS_ANDROID_BUGS },
   // These identify themselves as "cm_tenderloin", fun...
   // Done by HTC for HP I guess.
@@ -1745,9 +1759,10 @@
   { "Vizio", 0x0489, "VTAB1008", 0xe040, DEVICE_FLAGS_ANDROID_BUGS },
 
   /*
-   * Viewpia
+   * Viewpia, bq...
+   * Seems like some multi-branded OEM product.
    */
-  { "Viewpia", 0x2207, "DR", 0x0001, DEVICE_FLAGS_ANDROID_BUGS },
+  { "Various", 0x2207, "Viewpia DR/bq Kepler", 0x0001, DEVICE_FLAGS_ANDROID_BUGS },
 
   /*
    * Other strange stuff.
